pkg/types/aws/machinepool.go

```diff
@@ -48,13 +52,18 @@ type MachinePool struct {
 	// +kubebuilder:validation:MaxItems=10
 	// +optional
 	AdditionalSecurityGroupIDs []string `json:"additionalSecurityGroupIDs,omitempty"`
+
+	DataDisks []capav1beta2.Volume `json:"dataDisks,omitempty"`
```

Comment:
azure and vsphere has datadisks should we keep this for aws or use their "volumes"?